src/__tests__/__snapshots__/documenter.test.ts.snap

```diff
@@ -4415,6 +4415,15 @@ in the native control.",
       "optional": true,
       "type": "string",
     },
+    Object {
+      "description": "Specifies if the control is readOnly, which prevents the
+user from modifying the value, but does not prevent the value from
+being included in a form submission. A readOnly control is still focusable.
```

Comment:
From a content point of view "readOnly" is in camel case which would refer to the `readOnly` property but in a description as an adjective should this be "readonly" or "read-only"?

 (The internet is fairly divided how this is spelt)

Reply:
replaced every entry with `read-only`